app/controllers/provider/admin/dashboards_controller.rb

```diff
@@ -19,17 +18,9 @@ def show
     @unread_messages_presenter = unread_messages_presenter
   end
 
-  protected
```

Comment:
I don't see why, but I don't think it is harmful either 👍

Reply:
Well it is not used and Code Climate will complain